app/components/wallet/staking/dashboard/UserSummary.js

```diff
@@ -107,126 +117,138 @@ export default class UserSummary extends Component<Props, State> {
+              </div>
+            </div>
+          </>
+        ) : (
+          <LoadingSpinner small />
+        )}
+        <div className={styles.icon}>
+          <IconAda />
+        </div>
       </div>
     );
-  }
+  };
 
   getTotalRewards: void => Node = () => {
     const { intl } = this.context;
     const { totalRewards } = this.props;
     return (
-      <div className={styles.column}>
-        <div className={styles.header}>
-          <div className={styles.icon}>
-            <IconRewards />
-          </div>
-        </div>
-        <h3 className={styles.label}>
-          {intl.formatMessage(globalMessages.totalRewardsLabel)}:
-        </h3>
-        {totalRewards != null
-          ? (
-            <>
-              {totalRewards.unitOfAccount && (
-                <p className={styles.value}>
-                  {totalRewards.unitOfAccount.amount} {totalRewards.unitOfAccount.currency}
-                </p>
-              )}
+      <div className={classnames([styles.card, styles.mr20])}>
+        <h3 className={styles.label}>{intl.formatMessage(globalMessages.totalRewardsLabel)}:</h3>
+        {totalRewards != null ? (
+          <>
+            {totalRewards.unitOfAccount && (
               <p className={styles.value}>
-                {totalRewards.ADA} {this.props.meta.primaryTicker}
+                {totalRewards.unitOfAccount.amount} {totalRewards.unitOfAccount.currency}
               </p>
-              <div className={styles.sectionActions}>
-                {this.props.withdrawRewards != null && (
-                  <div
-                    className={classnames([styles.note, 'withdrawButton'])}
-                    role="button"
-                    tabIndex={0}
-                    onKeyPress={() => null}
-                    onClick={this.props.withdrawRewards}
-                  >
-                    {intl.formatMessage(globalMessages.withdrawLabel)}
-                  </div>
-                )}
-                <div
-                  className={styles.note}
-                  role="button"
-                  tabIndex={0}
-                  onKeyPress={() => null}
-                  onClick={this.props.openLearnMore}
-                >
-                  {intl.formatMessage(messages.note)}
-                </div>
-              </div>
-            </>
-          )
-          : (<LoadingSpinner small />)
-        }
+            )}
+            <p className={styles.value}>
+              {this.formatAdaValue(totalRewards.ADA)} {this.props.meta.primaryTicker}
+            </p>
+            <div className={styles.sectionActions}>
+              {this.props.withdrawRewards != null && (
+                <Button
+                  className='secondary'
+                  label={intl.formatMessage(globalMessages.withdrawLabel)}
+                  onClick={this.props.withdrawRewards}
+                  skin={ButtonSkin}
+                />
+              )}
+              {/* <div
```

Comment:
We're talking about re-adding this somehow in Slack. We can re-add it in a separate PR if you want though

Reply:
done